ocaml/xe-cli/newcli.ml

```diff
@@ -807,6 +817,9 @@ let main () =
         in
         let args = String.concat "\n" args in
         Printf.fprintf oc "User-agent: xe-cli/Unix/%d.%d\r\n" major minor ;
+        Option.iter
+          (Printf.fprintf oc "%s: %s\r\n" Http.Hdr.traceparent)
```

Comment:
this requires the new http_lib entry in the dune libraries section, which makes the binary larger only to use this constant. Alternatively, we could use the string "TRACEPARENT" here directly instead of using Http.Hdr.traceparent. What's the difference in binary size if we use the string directly instead of linking against http_lib?

Reply:
The difference is ~0.2M.

Reply:
Removed it here https://github.com/xapi-project/xen-api/pull/5633/commits/24681d75ee7e9b53648e2b54d62557009d9a506f. As the string is not likely to change in the future,  a 5% increase in the binary size doesn't seem worth.